shared/modules/persistence_module.go

```diff
@@ -25,10 +25,6 @@ type PersistenceModule interface {
 	// BlockStore maps a block height to an *coreTypes.IndexedTransaction
 	GetBlockStore() blockstore.BlockStore
 
-	// TreeStore manages atomic access to a set of merkle trees
-	// that compose the state hash.
-	GetTreeStore() TreeStore
```

Comment:
As we are removing this from the persistence context how can we access the `TreeStore` module from outside of the persistence package if needed?

Reply:
It can't be accessed from the outside, which is the desired outcome. Only the PersistenceContext should be exposed for external calls.